spec/01-unit/01-db/01-schema/06-routes_spec.lua

```diff
@@ -1512,7 +1512,13 @@ describe("routes schema (flavor = expressions)", function()
   end)
 
   it("http route still supports net.port but with warning", function()
+    local ngx_log = ngx.log
     local log = spy.on(ngx, "log")
+
+    finally(function()
+      ngx.log = ngx_log
```

Comment:
```suggestion
      ngx.log = ngx_log -- luacheck: ignore
```

Do this to avoid warning.

Reply:
done.